source/common/vue/form/elements/TextControl.vue

```diff
@@ -1,6 +1,7 @@
 <template>
-  <div v-bind:class="{ inline: inline === true, 'form-control': true }">
+  <div v-bind:class="{ inline: inline === true, 'form-control text-field': true }">
```

Comment:
I would separate the classes out; which makes it more verbose, yes, but also faster to parse that these are two classes